controllers/tasks.js

```diff
@@ -211,7 +212,7 @@ const assignTask = async (req, res) => {
   try {
     const { status, username } = req.userData;
 
-    if (status !== userStatusEnum[1]) {
+    if (status !== IDLE) {
```

Comment:
This would be more readable
```suggestion
    if (status !== userStatus.IDLE) {
```

Reply:
No, There is a better way. If you want to use more status suppose "active", you will do it like `userStatus.ACTIVE`. and for "ooo" you will use `userStatus.OOO`. So if we use an object to access property more than 2 times, we usually destructure the object.

Reply:
In RDS we task and user both have ACTIVE status. If we destructure, it'll be ambiguous that which entity the status belongs

Reply:
In that case. it makes sense not to destructure. I will make the change thank you.